docs/reference/method/MongoDBClient-listDatabases.txt

```diff
@@ -13,28 +13,27 @@ MongoDB\\Client::listDatabases()
 Definition
 ----------
 
-.. phpmethod:: MongoDB\\Client::listDatabases($options)
+.. phpmethod:: MongoDB\\Client::listDatabases()
 
-   Lists the available databases.
+   Returns information for all databases on the server.
 
    .. code-block:: php
 
       function listDatabases(array $options = []): MongoDB\Model\DatabaseInfoIterator
 
-   :phpmethod:`MongoDB\\Client::listDatabases` has the following parameters:
+   :phpmethod:`MongoDB\\Client::listDatabases()` has the following parameters:
```

Comment:
I generally suggest not using role-links to things defined on the current page, since they don't render the way humans would expect.